packages/components/src/unit-control/test/index.tsx

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+import { describe, expect, it } from '@jest/globals';
```

Comment:
The globals are still in scope, TypeScript just doesn't know about them. We need to add `jest` to the `types` array so that its types are loaded. We never import `jest` so its types are never loaded (unless we put them in the `types` array:

https://github.com/WordPress/gutenberg/blob/9b0e9e411df3db10772ddb67d44d380bd3007934/packages/components/tsconfig.json#L6

But this `tsconfig` file currently represents this entire project, it would make these globals become global which isn't accurate.

The way I've structured this before is to make 2 projects (2 tsconfigs):
- `tsconfig.json`: the application code, excludes tests files (something like `**/tests/` in this case?)
- `tsconfig.tests.json` - extends its sibling `tsconfig.json`, but only _includes_ tests files. This is where we include the jest and testing-library types.